### ISSUE TITLE
Visual Query SQL Connection Fails

### ISSUE THREAD

**Issue by jtherrien-accuraty** [2sic/2sxc#3285]
### I'm submitting a

[x] bug report => search github for a similar issue before submitting

### ...about

[x] Queries/Visual Query Designer

### Current Behavior / Expected Behavior

I am trying to use the 2sxc Query builder to access the SQL database, but using either the "Dnn SQL" or "SQL Data" source only throws the error "Error: Connection Problem" with the message "The ConnectionString property is empty / has not been initialized". However, if I go to the Sources tab it shows the query string being used is the correct one from web.config. Here is a screenshot of the error:
![image](https://github.com/2sic/2sxc/assets/62106757/2eb9c149-fcf0-490d-ad78-92a497be086d)

This actual SQL query doesn't seem to have any impact as I get the same error whether I am trying to access a custom table in the db, or a simple select * from the Dnn users table. Those same queries run as expected in the Dnn SQL console.

Run the SQL query.

Site originally had 2sxc v16 install and was upgraded to 17.1.6 before trying to add any queries through the visual designer.

### Your environment

* **2sxc version(s):** 17.1.6  
* **Browser:** Tested and error persists on Chrome and Firefox
* **DNN:** Only tested on 09.13.02
* **Hosting platform:** IIS
* **Language:**  English

**Comment by jeremy-farrance** [comment]
**This looks like a showstopper if upgrading a site with working Visual Queries!!**

I just tried this on a much older site with a working query, upgraded to v17.1.6 and the Query no longer works.
 
Added a new Visual Query, tried the regular "SQL Data" source instead (of DnnSql). Same issue.

Added a new Visual Query, added a sql data source, connected Default, changed the SQL command to simply: `SELECT * FROM dbo.Users` and did Save and Run. Same error as above. Also, in Sources I confirmed that the ConnectionString is the correct one and that the passw did not contain any "unfriendly" characters.

![image](https://github.com/2sic/2sxc/assets/653301/28d34685-c2a1-4130-9ed9-759731c3369f)
